cmake/compiler_features.cmake

```diff
@@ -271,4 +271,12 @@ MACRO(ROCKSDB_SET_DEFINTIONS)
 	if(HAVE_AUXV_GETAUXVAL AND NOT ROCKSDB_DISABLE_AUXV_GETAUXVAL)
 	  add_definitions(-DROCKSDB_AUXV_GETAUXVAL_PRESENT)
 	endif()
+
+        if(WITH_UBSAN)
+          add_definitions(-DROCKSDB_UBSAN_RUN)
```

Comment:
Do you have more context on how these flags are consumed?

Reply:
`ROCKSDB_UBSAN_RUN` appears to suppress UBSan warnings for misaligned pointers for the sites where RocksDB does that intentionally.